Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -203,15 +209,25 @@ void Mandelbrot::resize_event(GUI::ResizeEvent& event)
     m_set.resize(event.size());
 }
 
+void Mandelbrot::export_image(String const& export_path)
+{
+    auto png = Gfx::PNGWriter::encode(m_set.bitmap());
+    auto file = fopen(export_path.characters(), "wb");
```

Comment:
If the open fails we silently claim successful export.

Reply:
Fixed. :)